ui/src/pages/clients/ClientList.tsx

```diff
@@ -8,13 +8,17 @@ import { isoTimeToString } from "util/date";
 import usePanelParams from "util/usePanelParams";
 import EditClientBtn from "pages/clients/EditClientBtn";
 import DeleteClientBtn from "pages/clients/DeleteClientBtn";
+import Loader from "components/Loader";
+import Pagination from "components/Pagination";
+import { usePagination } from "util/usePagination";
```

Comment:
possible issue(?): client list after recent changes relies on `page_token` instead of the `page` parameter. If I'm not mistaken, this refers to the `page` parameter instead.

Reply:
I tested this with latest main branch and the parameter names now seem to be aligned, so we always use `page_token`. At least it worked for me locally. @shipperizer right?

Reply:
yes page_token is the one to use

not a good react reviewer but it seems you are still using page instead of page_token

this is what @BarcoMasile is pointing out I guess

Reply:
The page was only used in the browser, for the api request we'd use page_token. I changed that now, so the browser also uses page_token, so it is more consistent.